api_app/serializers.py

```diff
@@ -223,20 +220,6 @@ def plugins_to_execute(
 
         return plugins_to_execute
 
-    def filter_analyzers_requested(self, analyzers):
-        self.all_analyzers = False
-        if not analyzers:
-            analyzers = list(AnalyzerConfig.objects.all())
-            self.all_analyzers = True
-        return analyzers
-
-    def filter_connectors_requested(self, connectors):
-        self.all_connectors = False
-        if not connectors:
-            connectors = list(ConnectorConfig.objects.all())
-            self.all_connectors = True
-        return connectors
-
```

Comment:
Why did you remove this?

Reply:
The final configuration would be different

Reply:
Becase we don't want to takes all the available analyzer anymore, but we still want the flags `self.all_analyzers` and `self.all_connectors`. I replaced these functions with:
```
self.all_analyzers = (
            len(attrs["analyzers_requested"]) == AnalyzerConfig.objects.all().count()
        )
```

Reply:
On a second though, I think we should keep the chance to run all the available analyzers by default via API, in particular for retrocompatibility reasons

Reply:
Ok, i'm restoring the backend and maintain the filter only frontend-side